torchserve/datadog_checks/torchserve/check.py

```diff
@@ -0,0 +1,42 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.torchserve.inference_api_check import TorchserveInferenceAPICheck
+from datadog_checks.torchserve.management_api_check import TorchserveManagementAPICheck
+from datadog_checks.torchserve.openmetrics_check import TorchserveOpenMetricsCheck
+
+CONFIGURATION_OPTIONS = (
+    "openmetrics_endpoint",
+    "inference_api_url",
+    "management_api_url",
+)
+
+
+class TorchserveCheck(AgentCheck):
+    def __new__(cls, name, init_config, instances):
+        instance = instances[0]
+
+        configured_endpoint = [config for config in CONFIGURATION_OPTIONS if config in instance]
```

Comment:
Could you have this validation in `validators.py`?

Reply:
I don't think so because I do not use the ConfigMixin for this integration